escrow/src/data_structures.sw

```diff
@@ -0,0 +1,115 @@
+library data_structures;
+
+use core::ops::Eq;
+use std::{contract_id::ContractId, identity::Identity, option::Option};
+
+pub struct Arbiter {
+    /// Address identifying the arbiter
+    address: Identity,
+
+    /// The asset that the arbiter will be paid in upon resolution
+    asset: ContractId,
+
+    /// The quantity of asset to be taken as payment
+    fee_amount: u64,
+}
+
+impl Eq for Arbiter {
+    fn eq(self, other: Self) -> bool {
+        self.address == other.address && self.asset == other.asset && self.fee_amount == other.fee_amount
+    }
+}
+
+pub struct Asset {
+    /// Amount of asset the user must deposit
+    amount: u64,
+
+    /// The id used to identify the asset for deposit
+    id: ContractId,
+}
+
+pub struct Buyer {
+    /// Address identifying the buyer
+    address: Identity,
+
+    /// The asset that the user has currently deposited in the contract
+    asset: Option<ContractId>,
+
+    // Minor data duplication allows us to forego validating unique assets upon escrow creation
+    // otherwise the same asset with different values can be added which, if handled incorrectly,
```

Comment:
I don't agree with this suggestion because it seems to break the sentence incorrectly into two, furthermore, it appears to duplicate line 38 onto line 39 as well